santa.js

```diff
@@ -1,5 +1,17 @@
 function deliverPresents(houses) {
-  console.log('Delivering presents to ' + houses[0])
+  if (houses.lenght == 0) return
```

Comment:
Good job thinking of the case of an empty list, unfortunately this typo means it won't work.